pandas/tests/indexing/multiindex/test_setitem.py

```diff
@@ -15,114 +15,109 @@
 import pandas.core.common as com
 
 
+def assert_equal(a, b):
```

Comment:
this is shadowing a common function, would not do this

Reply:
you use a lambda

Reply:
it already exists, just moving it to a shared namespace

Reply:
right but i think should change it here

Reply:
sure.  follow-up?  the CI is gonna be slammed today